packages/react/src/lib/accessibility/types.ts

```diff
@@ -160,6 +160,7 @@ export interface AccessibilityDefinition {
   focusZone?: FocusZoneDefinition
   focusTrap?: FocusTrapDefinition
   autoFocus?: AutoFocusZoneDefinition
+  childrenBehavior?: Accessibility
```

Comment:
I would name it `childBehavior` as it is only one.

Reply:
I the sense of AccessibilityBehavior type, it can have only one child behavior. For e.g.

```
MenuBehavior= (props) => {
...
childBehavior: menuItemBehavior
}
```

Reply:
_It's not true_. `menuBehavior` has multiple siblings: `menuItemBehavior`, `menuDividerBehavior` & `submenuBehavior`. I had an offline conversation with @miroslavstastny and we agreed on the same thing that was proposed originally:
```js
menuBehavior= (props) => {
  // ...
  childBehaviors: {
     item: menuItemBehavior,
     divider: menuDividerBehavior,
  }
}
```

It allows to override behaviors for all child items and it's aligned with existing structure on `attributes` and `keyHandlers`. Please take one more look on this change.